ios/Classes/ObjectDetector.swift

```diff
@@ -256,10 +256,9 @@ public class ObjectDetector: Predictor {
             let prediction = results[i]
 
             var rect = prediction.boundingBox  // normalized xywh, origin lower left
-            
 
             if screenRatio >= 1 {  // iPhone ratio = 1.218
-            let transform = CGAffineTransform(scaleX: 1, y: -1).translatedBy(x: 0, y: -1)
+              let transform = CGAffineTransform(scaleX: 1, y: -1).translatedBy(x: 0, y: -1)
```

Comment:
By removing this line the ios version works correctly.

```suggestion
let transform = CGAffineTransform(scaleX: 1, y: -1).translatedBy(x: 0, y: -1) // remove this line.
```